cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -363,3 +363,21 @@ def get_or_create_content_group(store, course):
             content_group_configuration
         )
         return content_group_configuration
+
+    @staticmethod
+    def get_all_content_groups(store, course):
```

Comment:
I would make this get_all_partition_groups, or get_all_user_partition_groups, or even get_all_group_configuration_groups.

We need to settle whether or not an enrollment track group is a "content group". Enrollment Tracks are a type of UserPartition with scheme EnrollmentTrackPartitionScheme. Content Groups are a type of UserPartition with scheme CohortPartitionScheme. UserPartitions have Groups.